### DISCUSSION TITLE
Add big picture mode

### DISCUSSION THREAD

**Discussion by ZackStone** [iSoron/uhabits#611] [Feature Requests]
Create a feature to view the data like the image below.
It helps me to get some insights when I see the data in this format, grouping some habits.

![IMG_20200721_115603583](https://user-images.githubusercontent.com/9125490/88071348-13e4c680-cb4a-11ea-8f27-c9789a2d1030.jpg)

**Comment by iSoron** [comment]
Thank you for the suggestion, @ZackStone. What is the difference between this feature and the current app in landscape mode?

**Comment by vahidri** [comment]
Neat! Is this simply highlighting active days? Or is it sth like Convex Hull 😅 for the ticks?

(by convex hull I mean the circling around the adjacent chunks of ticks (or shadow-ticks = streak) )

**Comment by ZackStone** [comment]
@iSoron It's just because I would like to view something like a monthly report.
Even in landscape mode, I think it shows just a few days and tasks.
I thought in a "export as image/pdf" feature... or csv, but more easier to read than the csv that the app offers.
For example:
```csv
Tasks,2020-07-01,2020-07-02,2020-07-03,2020-07-04
Task 1,x,,,x
Task 2,x,,x,
Task 3,x,x,,x
```

**Comment by ZackStone** [comment]
@vramazani : I added these circles to analyze some patterns, that's why I would like to view a "monthly report".
That way I could get some insights, for example:
- When I drink coffee at night I don't sleep well
- I work better when I have lunch before 12 pm
- I feel good in this particular week... maybe because I did some workout in the morning every weekday

These 3 topics above would be the circles and little arrows (up and down == good and bad) in my personal notebook (the first picture here)

Can you understand the point? (my English is a little bit rusted 😝)

*I didn't know about the convex hull, but reading about it now, yeah, I think it makes sense!

**Comment by vahidri** [comment]
Great examples @ZackStone !
Astonishingly I thought of such relations between habits recently. Though I don't see how you'd infer those from simply grouping check marks.

I'm not a statistician, but I think there must be a _systematic_  approach; my initial intuition was calculating **Correlations**  between every habit pair and maybe sort or group those higher than a threshold (sth like 0.3 maybe) [using the csv output for now]

@iSoron , what do you think?
and Would this be a priority if we figure out the math?

(side note: yes, Correlation is Not Causation. it could be conducive tho)
